src/views/Home/components/FinancesLineChartCard/FinancesLineChartCard.stories.tsx

```diff
@@ -0,0 +1,72 @@
+import { createThemeModeVariants } from '@ses/core/utils/storybook/factories';
+import { financesLineChartCardData } from '@/views/Home/staticData';
+import FinancesLineChartCard from './FinancesLineChartCard';
+import type { Meta } from '@storybook/react';
+import type { FigmaParams } from 'sb-figma-comparator';
+
+const meta: Meta<typeof FinancesLineChartCard> = {
+  title: 'Fusion/Views/Home/FinancesLineChartCard',
+  component: FinancesLineChartCard,
+  parameters: {
+    chromatic: {
+      viewports: [375, 768, 1024, 1280, 1440],
+      pauseAnimationAtEnd: true,
+    },
+  },
+};
+
+export default meta;
+
+const variantsArgs = [
+  {
+    financesLineChartCard: {
+      tabButtonsTexts: financesLineChartCardData.tabButtonsTexts,
+    },
+  },
+];
+
+const [[LightMode, DarkMode]] = createThemeModeVariants(FinancesLineChartCard, variantsArgs);
+
+export { LightMode, DarkMode };
+
+LightMode.parameters = {
+  figma: {
+    component: {
+      375: {
+        component: 'https://www.figma.com/design/iLyzLutlWLu6Yf8tFdlM6T/Fusion%2FPowerhouse?node-id=5617-32011&m=dev',
+        options: {
+          componentStyle: {},
+          style: {},
```

Comment:
you should update the storybook-figma integration to correctly overlay the design over the implemented UI, otherwise, it is difficult for reviewers to do its job

Reply:
Done